ITR/data/base_providers.py

```diff
@@ -626,9 +628,10 @@ def _winsorize(self, historic_intensities: pd.DataFrame) -> pd.DataFrame:
                                                         numeric_only=False),
                     axis='columns'
                 )
-                if uncertain_intensities.values.sum() != 0:
-                    uwinsorized = winsorized.apply(lambda x: PA_(unp.uarray(x.values.data, uncertain_intensities[x.name].values), dtype=units[x.name]))
-                    return uwinsorized
+                if ITR.HAS_UNCERTAINTIES:
+                    if uncertain_intensities.values.sum() != 0:
+                        uwinsorized = winsorized.apply(lambda x: PA_(ITR.uarray(x.values.data, uncertain_intensities[x.name].values), dtype=units[x.name]))
+                        return uwinsorized
```

Comment:
even w/o `HAS_UNCERTAINTIES`, we may have a "sum() != 0", which we skip now.
as a result, results differ depending on if we have the pint support for uncertainties or not

Reply:
What we are skipping is whether we need to do winsorization to UFloat values.  If there are no such values, we simply winsorize the np.float64 values, which is the fall-through condition.